js/script.js

```diff
@@ -22,16 +23,213 @@ const greenquadrant = document.querySelector("#greenquad");
 const redquadrant = document.querySelector("#redquad");
 const bluequadrant = document.querySelector("#bluequad");
 const yellowquadrant = document.querySelector("#yellowquad");
+const maxRound = 20;
 
-// start the GAME
+// Click the power button to activate the game
 
 powerOn.addEventListener('click', (event) => {
   if (powerOn.checked == true) {
-    power = true
-    countTurn.innerHTML = "_";
+    powerSelection = true
+    countTurn.innerHTML = "--";
+    startButton.disabled = false;
+    strictOn.disabled = false
 
   }else {
-    power = false
+    powerSelection = false
+    startButton.disabled = true;
+    strictOn.disabled = true;
     countTurn.innerHTML = "";
+    // clearFlash();
+    clearInterval(intervalId);
+    strictOn.checked = false;
+  }
+});
+
+// Click the start button to start the game
+startButton.addEventListener('click', (event) => {
+  if (powerSelection == true) {
+    playNewGame();
+    }
+});
+
+
+// Enable strict button to start strict mode.
+strictOn.addEventListener('click',(event) => {
+  if(strictOn.checked == true){
+    strictMode = true;
+    strictPlay();
+  }
+});
+
+
+// Function to play the game
+function playNewGame () {
+// reset variables
+  winTheGame = false;
+  userSeq = [];
+  sound = true;
+  intervalId = 0;
+  quadFlash = 0;
+  playTurn = 1;
+  countTurn.innerHTML = 1;
+  goodPlay = true;
+  startGame = true;
+  compPlay();
+
+};
+
+function click(num){
+  const [audio, quad, lite] = getAttributes(num)
+  if(sound){
+    audio.play()
+  }
+  quad.classList.add(lite)
+  setTimeout(function(){
+    quad.classList.remove(lite)
+  }, 500)
+}
+
+function getAttributes(num){
```

Comment:
Instead of doing our long if-else from before - `if (currentPlay == 1)`... and stuff like that... we dedicate a function to get all the attributes of a particular number. we pass the number as argument to the click function and it does everything we need. that's why I told you about giving responsibilities to functions so your code can be simpler.